src/components/profile/userProfile.jsx

```diff
@@ -5,54 +5,86 @@ import Box from '@material-ui/core/Box';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import { Divider } from '@material-ui/core';
+import { logout, history } from '../../actions/signInAction';
+import Navlinks from './Nav.links.jsx';
 
 export class UserProfile extends Component {
 	// getting a loged-in user information
 	UNSAFE_componentWillMount() {
 		this.props.GetUserProfile();
 	}
+	state = {
+		logout: false,
+	};
 
 	render() {
 		const { userProfileInfo, socketManagement, notifications } = this.props;
 		return (
 			<>
 				{userProfileInfo && (
-					<Box width='100%' style={{ height: '100%' }}>
-						{socketManagement(userProfileInfo.id)}
-						<Link to='/profile' style={{ textDecoration: 'none' }}>
-							<Box width='100%' height='110px'>
-								<Typography align='center'>
-									<img
-										style={{
-											width: '30%',
-											height: '70px',
-											borderRadius: '50%',
-											objectFit: 'cover',
-										}}
-										src={userProfileInfo.profileImage}
-										alt=''
-									/>
-								</Typography>
-							</Box>
-							<Box fontStyle={2} alignSelf={3}>
-								<Typography
-									align='center'
-									style={{ fontSize: '18px', color: 'gray' }}
-								>
-									{userProfileInfo.firstName + ' ' + userProfileInfo.lastName}
-								</Typography>
-							</Box>
-							<Box marginTop={1}>
-								<Typography
-									align='center'
-									style={{ fontSize: '18px', color: 'gray' }}
-								>
-									{userProfileInfo.email}
-								</Typography>
-							</Box>
-						</Link>
-					</Box>
+					<>
+						<Box width='100%' style={{ height: '100%' }}>
+							{socketManagement(userProfileInfo.id)}
+							<Link to='/profile' style={{ textDecoration: 'none' }}>
+								<Box width='100%' height='80px'>
+									<Typography align='center'>
+										<img
+											style={{
+												width: '30%',
+												height: '70px',
+												borderRadius: '50%',
+												objectFit: 'cover',
+											}}
+											src={userProfileInfo.profileImage}
+											alt=''
+										/>
+									</Typography>
+								</Box>
+								<Box fontStyle={2} alignSelf={3}>
+									<Typography
+										align='center'
+										style={{ fontSize: '18px', color: 'gray' }}
+									>
+										{userProfileInfo.firstName + ' ' + userProfileInfo.lastName}
+									</Typography>
+								</Box>
+								<Box marginTop={1}>
+									<Typography
+										align='center'
+										style={{ fontSize: '18px', color: 'gray' }}
+									>
+										{userProfileInfo.email}
+									</Typography>
+								</Box>
+							</Link>
+						</Box>
+						<Navlinks />
+						<Box marginTop={1}>
+							<ListItem
+								id='dashboard'
+								button
+								onClick={() => {
```

Comment:
@Jajabenit250 it's always better to write a handle function rather than defining an anonymous function on the eventListener. Kindly update.